### DISCUSSION TITLE
"export 'stream' (imported as 'openpgp') was not found in 'openpgp'

### DISCUSSION THREAD

**Discussion by ehte92** [openpgpjs/openpgpjs#1365] [Q&A]
Getting a warning in the latest version when I am using the openpgp.stream and also getting a type error

`TypeError: Cannot read property 'readToEnd' of undefined`

I am using the following to read the stream to the end and store it in a variable.

`const plainStr = await openpgp.stream.readToEnd(decryptBlob.data);`

Is there any new changes in the new version?

**Comment by larabr** [comment]
Hi @ehte92 ,  `openpgp.stream` was unexported in 5.0.0-2, see https://github.com/openpgpjs/openpgpjs/pull/1291 .

**Comment by AndreasGassmann** [comment]
It looks like you can use the `@openpgp/web-stream-tools` package. I'm not sure if this is intended, and there are also no typescript typings available, but it seems to work.

```
import { readToEnd } from '@openpgp/web-stream-tools'
```
